### PR TITLE
Fix: issue4179 translate login and signup in French

### PR DESCRIPTION
**Pull request by celcarpe**

### CODE REVIEW

**Comment by Navaneeth-pk** [comment]
@celcarpe please revert the changes made in `package-lock.json` and `package.json`.

**Comment by celcarpe** [comment]
I messed up my reverts sorry, should be okay now

**Review by withshubh — DISMISSED** [review]

File: frontend/assets/translations/fr.json [hunk 1]
```diff
@@ -71,24 +71,24 @@
...
+        "signIn":"Se connecter",
+        "signUp":"S'inscrire",
+        "createToolJetAccount":"Créer un compte ToolJet",
+        "enterBusinessEmail":"Entrer votre adresse e-mail professionnelle",
+        "alreadyHaveAnAccount":"Vous avez déjà un compte ?",
+        "password":"Mot de passe",
+        "showPassword":"montrer le mot de passe",
+        "loginTo":"Se connecter à",
+        "yourAccount":"votre compte",
+        "noLoginMethodsEnabled":"Aucune méthode de connexion est autorisée pour cet espace de travail",
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:88]:
```suggestion
        "noLoginMethodsEnabled":"Aucune méthode de connexion n'est activée pour cet espace de travail",
```

File: frontend/assets/translations/fr.json [hunk 2]
```diff
@@ -71,24 +71,24 @@
...
+        "emailConfirmLink":"Veuillez vérifier vos mails pour le lien de confirmation",
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:89]:
```suggestion
        "emailConfirmLink":"Veuillez vérifier votre e-mail pour le lien de confirmation",
```

File: frontend/assets/translations/fr.json [hunk 3]
```diff
@@ -71,24 +71,24 @@
...
-        "showPassword":"show password",
-        "loginTo":"Login to",
-        "yourAccount":"your account",
-        "noLoginMethodsEnabled":"No login methods enabled for this workspace",
-        "emailConfirmLink":"Please check your email for confirmation link",
-        "newPassword":"New Password",
-        "passwordConfirmation":"Password Confirmation"
+        "forgotPassword":"Mot de passe oublié",
+        "emailAddress":"Adresse e-mail",
+        "enterEmail":"Entrer email",
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:76]:
```suggestion
        "enterEmail":"E-mail",
```

**Review by roiLeo — COMMENTED** [review]
I've added notes

**Comment by withshubh** [comment]
Hey @celcarpe 
Please make the changes suggested by @roiLeo

**Comment by celcarpe** [comment]
I did the changes as suggested, lmk if something else has to be changed

**Review by withshubh — APPROVED** [review]

**Pull request merged**